azure/services/scalesets/scalesets_test.go

```diff
@@ -635,14 +637,15 @@ func getFakeSkus() []compute.ResourceSku {
 	return []compute.ResourceSku{
 		{
 			Name: to.StringPtr("VM_SIZE"),
+			ResourceType: to.StringPtr(string(resourceskus.VirtualMachines)),
```

Comment:
Without specifying the `ResourceType`, the capability was not found in the tests.